english/marshmallow.md

```diff
@@ -1,4 +1,4 @@
-This is a story to look for your own, to [fight your way out of this](fight-out/fight-out.md) or to calm down 
+[This](this/new-story.md) is a story to look for your own, to [fight your way out of this](fight-out/fight-out.md) or to calm down 
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.